packages/@coorpacademy-components/src/molecule/setup-cohort-item/popinstyle.css

```diff
@@ -0,0 +1,50 @@
+@value colors: "../../variables/colors.css";
+@value dark from colors;
+
+.disable {
+  top: calc(-70vh);
```

Comment:
```suggestion
  top: 0px;
```

Reply:
don't use `calc` is partially supported by browsers

Reply:
It's Done